### PR TITLE
Update mentions of key_bindings and mouse_bindings in the manual

### PR DESCRIPTION
**Pull request by proski**

### CODE REVIEW

File: extra/man/alacritty.5.scd [hunk 1]
```diff
@@ -642,12 +642,12 @@ of the configuration file.
 
 		Mouse button which needs to be pressed to trigger this binding.
 
-	*action* <key_bindings.action> | "ExpandSelection"
+	*action* <keyboard_bindings_action> | "ExpandSelection"
```
Comment (kchibisov) [inline extra/man/alacritty.5.scd:645]:
```suggestion
	*action* <keyboard.bindings_action> | "ExpandSelection"
```

**Review by kchibisov — CHANGES_REQUESTED** [review]

**Review by kchibisov — APPROVED** [review]

**Review by chrisduerr — CHANGES_REQUESTED** [review]
I think we should document keyboard/mouse as separate sections for the sake of consistency.

**Comment by proski** [comment]
@chrisduerr Please explain. The documentation is already using separate sections. The only overlap is in the actions. I can move actions to a separate section.

**Comment by chrisduerr** [comment]
`keyboard` and `mouse` are TOML tables and should be in dedicated sections. No other section is collapsed the way you did it.

**Comment by kchibisov** [comment]
@proski if it's not clear what is wanted here, the mouse and bindings sections are both in the `# Bindings`, however what was suggested instead is that `mouse_bindings` will be moved to `# Mouse` section and will be called not `mouse.bindings`, but just `bindings`.

The same for keyboard, however the `# Bindings` should be renamed to `# Keyboard` at this point and `key_bindings` will become just `bindings`.

That should be just moving sections around.

The `# Keyboard` section should have entry docs like `# Mouse` one.

**Comment by proski** [comment]
The manual sections reflect the top-level sections of the TOML file now.

**Review by kchibisov — APPROVED** [review]

**Review by chrisduerr — APPROVED** [review]

**Pull request merged**